ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-release-4.18__images.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
```

Comment:
Should we use 1.22 https://github.com/openshift/cluster-api-actuator-pkg/blob/086923b9d3fd72a50a6c1587f8dc1189cae48df4/go.mod#L3 ?

Reply:
Thanks @damdo we should update https://github.com/openshift/cluster-api-actuator-pkg/blob/master/Dockerfile.ci as well?

Reply:
Yeah sounds reasonable. Thank you!